modules/cloud-router-connection/outputs.tf

```diff
@@ -0,0 +1,6 @@
+output "primary_connection_result" {
```

Comment:
We should name this `primary_connection_id` because that's what we're actually outputting from the module. The secondary one should follow the same pattern.